wrappers/core18.go

```diff
@@ -137,7 +137,7 @@ func AddSnapdSnapServices(s *snap.Info, inter interacter) error {
 		return nil
 	}
 
-	sysd := systemd.New(dirs.GlobalRootDir, systemd.SystemMode, inter)
+	sysd := systemd.NewUnderRoot(dirs.GlobalRootDir, systemd.SystemMode, inter)
```

Comment:
can you add a comment why we are using NewUnderRoot here? is it used during preseed? or something else?

Reply:
I think it's actually fine to use systemd.New here as well. This affects services from snapd from snap.